elixir/1.7/Makefile

```diff
@@ -14,7 +14,7 @@ require-%:
 
 build: require-VERSION require-PLATFORM
 	docker build \
-		-t ${PLATFORM}:${VERSION} \
+		-t vbt-${PLATFORM}:${VERSION} \
```

Comment:
please extract to PREFIX env var or use the existing REPO var.

Reply:
Changed